data/os/Debian/7.yaml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
we don't support debian 7 anymore (and 8 will be dropped in the next days if I finde some free time), please don't add the yaml file for debian 7.